src/Illuminate/Notifications/Channels/HttpChannel.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Illuminate\Notifications\Channels;
+
+use GuzzleHttp\Client as HttpClient;
+
+abstract class HttpChannel
+{
+    /**
+     * The HTTP client instance.
+     *
+     * @var \GuzzleHttp\Client
+     */
+    protected $http;
+
+    /**
+     * Create a new HTTP channel instance.
+     *
+     * @param  \GuzzleHttp\Client  $http
+     * @return void
+     */
+    public function __construct(HttpClient $http)
+    {
+        $this->http = $http;
+    }
+}
```

Comment:
On that logic, I see this abstract class as pointless then. Makes no sense to have this if only for a constructor. Infact, I think this is is an antipattern.